services/py-api/src/server/routes/utility_routes.py

```diff
@@ -17,3 +19,8 @@ async def ping(
     response: Response, handler: UtilityHandlers = Depends(create_utility_handler)
 ) -> PongResponse | ErrResponse:
     return await handler.ping_services(response)
+
+#Test encode function
+@utility_router.get("/testjwt", responses={200: {"model": PongResponse}, 503: {"model": ErrResponse}})
+async def testJWT() -> Any:
+    return await JwtUtility.encode({"str":"fdsafsad"})
```

Comment:
Remove this, we don't need a separate route to test the jwt functionality.

I suggest @plamenapavlova write an "integration test" for this, as writing a unit test is not going to do much work for us. (we want to test the real behavior of the jwt utils you have written)

What the integration test should do:

Encode some info and then try to decode it -> it should return success
Encode invalid info and check if returns an error
Decode an expired jwt token (you could have it hardcoded in a const)
Decode a jwt token which is signed with a different secret (you could have it the test-secret in a const)
Decode a malformed JWT

Reply:
Why are you dismissing unit tests especially in the context of an auth library? All of the cases you just explained can be validated with unit tests.

Reply:
Fixed in 2cf34e41919919fce21b30e78d7ee0dfaac03ca9